### ISSUE TITLE
User should have to confirm before updating profile photo

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#2377]
Right now when a user updates a profile photo it gets sent to everyone's device. If they pick the wrong image in the file picker by mistake there will be unpleasant consequence of having that image sent to the devices of everyone in the community. We should have some confirmation for this.

**Comment by jgaylor** [comment]
@holmesworcester In both 'change logo' for org and profile designs there is a step to preview the image before continuing. This can also double as an opportunity to refine the image by providing the ability to crop, scale, and rotate. We shouldn't send the image to everyone's device until the user submits this. This offers an opportunity to cancel and pick a different image.

Are you saying we should have an additional confirmation after the preview?

**Comment by holmesworcester** [comment]
Let's just use the existing designs here where there is a stage for crop, and note that the current implementation does not match the designs.

TL;DR: we need to add the cropping phase, and upload should not happen until after that.

**Comment by jgaylor** [comment]
Latest [Profiles](https://www.figma.com/design/BnANosC1KGMUvm8oU2Dr0i/Profiles?node-id=799-7625&t=GmRCGcHCQLdv87v1-1) Figma
